introduction.md

```diff
@@ -1 +1,14 @@
 my favorite path is HTML/CSS
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?